### ISSUE TITLE
Update IPv6 prefix-length and prefix-id for Google Fiber configs

### ISSUE THREAD

**Issue by burkasaurusrex** [stevejenkins/UBNT-EdgeRouter-Example-Configs#24]
It seems as though Google Fiber has switched from provisioning /56 IPv6 addresses to /64. Would be great to update the configs accordingly.

I'm no networking expert, but I updated the prefix-length to /64 and deleted the prefix-ids, and my ERPOE-5 is now issuing IPv6 addresses again.

**Comment by stevejenkins** [comment]
Thanks. I noticed this too. Bummer. Getting /56 was awesome. :)

I'll update!

**Comment by stevejenkins** [comment]
Fixed with c5bb7af .

**Comment by burkasaurusrex** [comment]
I think you may need to clear out the prefix-ids as well? I kept receiving this error when I tried to set it to anything: `prefix-id must be less than :: for prefix /64`

**Comment by wastrachan** [comment]
@stevejenkins I'm not sure if Google has gone back on a previous decision... but I lost my PD the other day and spent several days debugging the issue, best I could. 

I was able to successfully request a `/60` but SLAAC still wasn't working within my network. The only change that got things working again... You guessed it... was increasing my prefix size back to a `/56`. I can't account for the lack of SLAAC when requesting a `/60`, Google did acknowledge the request and assign a /60, *and* I saw RA's in my network, but none of my devices were participating. If I ever come into more knowledge on ipv6 and can figure out what was wrong with the RA's I'll update. 

But in the meantime: consider this a report that `/64` PD request isn't working for me any longer, and I had to request a `/56` to restore connectivity.
